packages/gitbook/src/components/AIActions/AIActions.tsx

```diff
@@ -0,0 +1,188 @@
+import { Button } from '@/components/primitives/Button';
+import { DropdownMenuItem } from '@/components/primitives/DropdownMenu';
+import { tString, useLanguage } from '@/intl/client';
+import { Icon, type IconName, IconStyle } from '@gitbook/icons';
+import { useState } from 'react';
+
+type AIActionType = 'button' | 'dropdown-menu-item';
+
+export function OpenDocsAssistant(props: { type: AIActionType }) {
+    const { type } = props;
+    const chatController = useAIChatController();
+    const chat = useAIChatState();
+    const language = useLanguage();
+
+    return (
+        <AIActionWrapper
+            type={type}
+            icon={<AIChatIcon />}
+            label="Ask Docs Assistant"
+            description="Ask our Docs Assistant about this page"
+            onClick={() => {
+                // Open the chat if it's not already open
+                if (!chat.opened) {
+                    chatController.open();
+                }
+
+                // Send the "What is this page about?" message
+                chatController.postMessage({
+                    message: tString(language, 'ai_chat_suggested_questions_about_this_page'),
+                });
+            }}
+        />
+    );
+}
+
+export function CopyMarkdown(props: { markdown: string; type: AIActionType }) {
+    const { markdown, type } = props;
+    const [copied, setCopied] = useState(false);
+
+    // Close the dropdown menu manually after the copy button is clicked
+    const closeDropdownMenu = () => {
+        const dropdownMenu = document.querySelector('div[data-radix-popper-content-wrapper]');
+
+        // Cancel if no dropdown menu is open
+        if (!dropdownMenu) return;
+
+        // Dispatch on `document` so that the event is captured by Radix's
+        // dismissable-layer listener regardless of focus location.
+        document.dispatchEvent(new KeyboardEvent('keydown', { key: 'Escape', bubbles: true }));
+    };
+
+    return (
+        <AIActionWrapper
+            type={type}
+            icon={copied ? 'check' : 'copy'}
+            label={copied ? 'Copied!' : 'Copy for LLMs'}
+            description="Copy page as Markdown"
+            onClick={(e) => {
+                e.preventDefault();
+
+                if (!markdown) return;
+                navigator.clipboard.writeText(markdown);
+                setCopied(true);
+
+                setTimeout(() => {
+                    if (type === 'dropdown-menu-item') {
+                        closeDropdownMenu();
+                    }
+
+                    setCopied(false);
+                }, 2000);
+            }}
+        />
+    );
+}
+
+export function ViewAsMarkdown(props: { markdownPageUrl: string; type: AIActionType }) {
+    const { markdownPageUrl, type } = props;
+
+    return (
+        <AIActionWrapper
+            type={type}
+            icon={<MarkdownIcon className="size-4 fill-current" />}
+            label="View as Markdown"
```

Comment:
```suggestion
            label={tString(language, 'view_page_markdown')}
```

and then add to `en.ts` and other translations:
```
view_page_markdown: 'View as Markdown',
```